src/proto_conversions.cc

```diff
@@ -138,6 +138,42 @@ const std::string& Options_Action_Enum_Name(const Options::Action action) {
   return i == actions.cend() ? empty : i->second;
 }
 
+bool Location_Type_Enum_Parse(const std::string& type, Location::Type* t) {
```

Comment:
we were manually doing this before so i added these standard methods for going between strings and enums. choosing `k` named variables keeps us from using the built in protobuf implementations although older versions of protobuf didnt have these methods either in the message lite header